config/local-multi/api-catalog-service.yml

```diff
@@ -1,12 +1,8 @@
 environment:
-    discoveryLocations: https://localhost2:10021/eureka/,https://localhost:10011/eureka/
     dsIpAddress: 0.0.0.0
     eurekaPassword: password
     eurekaUserId: eureka
     gatewayHostname: https://localhost:10010
```

Comment:
Could you please update the rest of the variables prefixed with environment? We want to make it consistent - prefixed with apiml.service

Reply:
Changes made. All keys under environment moved to under `apiml.service` . Please have a look. Thanks